docs/markdown/Getting Started/getting-started/prerequisites.md

```diff
@@ -42,9 +42,7 @@ System-specific notes
 
 > 📘 Apple Silicon (M1/M2) support
 >
-> If you have Python code, you may need to [set your interpreter constraints](doc:python-interpreter-compatibility) to Python 3.9+, as many tools, such as Black, will not install correctly when using earlier Python versions.
->
-> When running in Docker you will need to set `--no-watch-filesystem --no-pantsd`. (Although we don't recommend permanently setting this, as these options are crucial for performance when iterating.)
```

Comment:
should the mention of py3.9+ incompatibility with m2/m2 be moved somewhere within the python section?

Reply:
I think I'd rather leave it out, since it's either not a Pants-specific issue, or it's a Pants bug. Basically, if it works outside Pants it should work inside Pants. If it doesn't, that's not really a Pants thing?

Reply:
that's true! but I think you'll keep getting people opening issues on github/slack about it if people with apple silicon products try out pants and run into the above common errors